components/dialog/dialog.html

```diff
@@ -70,6 +70,10 @@
         border-bottom: var(--border);
     }
 
+    #btnToday:hover, #btnDatePickerClose:hover {
+        background-color: var(--grey-s3) !important;
```

Comment:
Be careful of !important its use should only be in very specific scenarios prefer specificity
If your inside a shadow root its a bit safer too